meetings/2018-12-12.md

```diff
@@ -0,0 +1,110 @@
+* **GitHub Issue**: https://github.com/nodejs/TSC/issues/639
+
+## Present
+
+* Anna Henningsen @addaleax (TSC)
+* Сковорода Никита Андреевич @ChALkeR (TSC)
+* Colin Ihrig @cjihrig (TSC)
+* Joyee Cheung @joyeecheung (TSC)
+* Gabriel Schulhof @gabrielschulhof (TSC)
+* Matteo Collina @mcollina (TSC)
+* Michael Dawson @mhdawson (TSC)
+* Michaël Zasso @targos (TSC)
+* Sakthipriyan Vairamani @thefourtheye (TSC)
+
+## Agenda
+
+### Announcements
+
+* No annoucements this week
+
+### nodejs/node
+
+* cli: add --max-http-header-size flag [#24811](https://github.com/nodejs/node/pull/24811)
+  * Added to agenda for visibility.
+  * Plan is to land and backport sooner than later as people were affected by change in the
+    community release and this provides a work around.
+  * Removed from agenda for now.
+
+* Fixing child_process module to check values passed strictly to the options object [#24267](https://github.com/nodejs/node/pull/24267)
+  * controversial as there is disagreement in the issue as to whether this should land.
+  * Back to github and invite Liran to next TSC meeting to provide a deeper dive and then
+    we can decide on next steps.
+    
+### nodejs/nodejs.org
+
+* Re-enstate link to NiM in debug documentation. [#1923](https://github.com/nodejs/nodejs.org/issues/1923)
+  * Push back to github and try to be up to speed to discuss in more detail next
+    week.
+
+### nodejs/TSC
+
+* Tracking issue for updating TSC on Board Meetings [#476](https://github.com/nodejs/TSC/issues/476)
+  * Board meeting next week
+
+* Strategic Initiatives - Tracking Issue [#423](https://github.com/nodejs/TSC/issues/423)
+  * Modules - Quite a bit of discussion happening right now in two break out proposals
+    https://github.com/GeoffreyBooth/node-import-file-specifier-resolution-proposal
+
+    A proposal to utilize metadata to determine the module type of a package allowing
+    for us to support importing both esm and cjs modules.
+
+    https://github.com/jkrems/proposal-pkg-exports/
+
+    An alternative approach to the current Node.js resolution algorithm. Meta Data in  
+    package.json would be used to identify exports. Compatible with [import-map
+    proposal](https://github.com/domenic/import-maps/)
+  
+  * N-API - a few pull requests, discussed content of what might be in N-API version 4, 
+    content would only be thread-safe as others.
+
+  * Workers - Gabriel has been working with Anna to better integrate support for
+    native modules. Allows for module unloading, remaining blocking issue is crash on
+    AIX, Gabriel investigating.
+
+  * Governance - we started a project board to track responsibilities of various
+    stakeholders within the proposed foundation. We will be using this artifact to
+    help determine how many Top-Level committees there will be and what they
+    will be responsible for. https://github.com/nodejs/bootstrap/projects/1
+
+  * Open Web Standards https://github.com/nodejs/node/pull/21551
```

Comment:
likely need a space between title and the link

Reply:
@MylesBorins fixed.